core/nuxeo-resource.js

```diff
@@ -251,7 +251,9 @@ import './nuxeo-connection.js';
         options.headers['nx-es-sync'] = true;
       }
       // add support for plain text responses by default
-      options.headers.accept = 'text/plain,application/json';
+      if (!Object.prototype.hasOwnProperty.call(options.headers, 'accept')) {
+        options.headers.accept = 'text/plain,application/json';
+      }
```

Comment:
can't we just do:
```suggestion
      options.headers.accept = options.headers.accept || 'text/plain,application/json'
```
?

Reply:
We can! It would look cleaner and I don't think we have a use that requires for explicitly checking the present for the property on the object?